[NET-863] Add note about Cilium reqs for NPM (#8881)
DataDog/documentation: content/en/network_performance_monitoring/installation.md

There is an exemption to the 4.4.0+ kernel requirement for [CentOS/RHEL 7.6+][2]. The [DNS Resolution][3] feature is not supported on CentOS/RHEL 7.6.

> Network Performance Monitoring is compatible with Cilium installations, provided the following requirements are met:

Reviewer: ```suggestion
Network Performance Monitoring is compatible with **Cilium** installations, provided the following requirements are met:
```
Motivation:
The number of requirements + exceptions is starting to get lengthy + hard to parse - I would bold "Cilium", "provisioning systems", and "platform" above for readability.

> Network Performance Monitoring is compatible with Cilium installations, as long as the following requirements are met:

Reviewer 2: ```suggestion
Network Performance Monitoring is compatible with Cilium installations, provided the following requirements are met:
```

1) Cilium version >= 1.6, and

> 2) Kernel version >= 4.19.57, >= 5.1.16, or >= 5.2

Reviewer 3: Doesn't >=4.19.57 imply >=5.1.16 and >=5.2?

Reviewer 2: Depends on if you interpret the numbers as numbers or as version numbers :) For this reason, I'm going to suggest turning the >= into english words. Something like "Cilium versions later than 1.6" and "Kernel versions later than 4.19.57, 5.1.16, or 5.2"

Author: I think you have that backwards ( >=5.2 implies >= 5.1.16 and >= 4.19.57)?
But either way, the way I understood it is that you're fine if you a newer kernel release 5.X, X>=2, but if you want to run an older release 4.19 and 5.1 are still supported as long as the subversions are >=57/16 (respectively)
I'm only guessing though, those requirements came from [here](https://docs.cilium.io/en/v1.6/install/system_requirements/#advanced-features-and-required-kernel-version) and there really isn't much more elaboration :/

Reviewer 3: Either way, I interpret 5.1.16 as greater than 4.19.57

Author: I think it's supposed to mean you can run a newer kernel release 5.X, X>=2, but there's also support for 4.19.X and 5.1.X as long as X>=57/16 (respectively)

I'm only guessing though, the kernel version requirement comes from [here](https://docs.cilium.io/en/v1.6/install/system_requirements/#advanced-features-and-required-kernel-version) and there isn't much of an explanation

Reviewer: Wdyt about shortening to 
"Network Performance Monitoring is compatible with Cilium installations v1.6+ with kernel versions later than X, Y, and Z."

Author: Since the kernel version requirements are a bit lengthy, I think it's easier to read when the requirements are listed separately

**Note**: Datadog does not currently support Windows and macOS platforms for Network Performance Monitoring.

> The following provisioning systems are supported:

Reviewer: ```suggestion
The following **provisioning systems** are supported:
```
